openproblems/tasks/spatial_decomposition/datasets/sc_to_sp.py

```diff
@@ -0,0 +1,41 @@
+from ....data.pancreas import load_pancreas
+from ....tools.decorators import dataset
+from ._sc_to_sp_utils import generate_synthetic_dataset
+from ._sc_to_sp_utils import get_pancreas_integer
```

Comment:
TBH since this is specific to the pancreas dataset, I think both this helper and the dataset functions should be moved to `pancreas.py`